Sources/MountebankSwift/Models/Imposter/Imposter.swift

```diff
@@ -20,7 +20,7 @@ public struct Imposter: Codable, Equatable {
         /// - Parameters:
         ///   - allowCORS: If true, mountebank will allow all Cross-Origin Resource Sharing preflight
         ///     requests on the imposter
-        case http(allowCORS: Bool? = false)
+        case http(allowCORS: Bool? = nil)
```

Comment:
I've changed this because MB does never return this field when getting an imposter. This leads to bugs in our tests. I will file an issue @ mountebank for this later, but I think for now this is fine and in line with our other default properties.